tests/integration/security/testdata/authz/v1beta1-ingress-gateway.yaml.tmpl

```diff
@@ -16,6 +16,24 @@ spec:
             hosts: ["internal.company.com"]
         - operation:
             paths: ["/private"]
+    - from:
```

Comment:
can you update the comments in the beginning of this file, it's no longer only testing internal.company.com and /private.